groups/models.py

```diff
@@ -143,3 +143,10 @@ class TextComment(BaseComment):
 
     class Meta(BaseComment.Meta):
         pass
+
+
+class FileComment(BaseComment):
+    file = models.FileField()
+
+    class Meta(BaseComment.Meta):
+        pass
```

Comment:
I think you don't need the `Meta` class as it would be inherited from the parent `BaseComment`